docs: Add instructions for connecting to Cloud SQL as the `firezone` user (#7527)
CloudFire-LLC/cloudfire: elixir/README.md

gcloud auth application-default login
```

### Connecting to Cloud SQL instance as the `firezone` user

> Some operations like DROPing indexes or tables require you to connect as the table owner, which in our case is the `firezone` user.

Author: ```suggestion
Some operations like DROPing indexes to recreate them require you to connect as the table owner, which in our case is the `firezone` user.
```
